### DISCUSSION TITLE
Realtime updates not being sent in specific project

### DISCUSSION THREAD

**Discussion by CactusPuppy** [supabase/supabase#9340] [Questions] [answered]
After following the instructions provided in https://supabase.com/docs/guides/api#realtime-api, having that not work, trying some suggestions in [this discussion (#428)](https://github.com/supabase/supabase/discussions/428), and even making the (questionable) decision to run [this SQL query in the SQL editor](https://github.com/supabase/realtime/issues/265#issuecomment-1196604286), I am able to establish a websocket connection which doesn't immediately close, but I do not receive any websocket messages regarding database updates.

The code I'm running (using the Supabase JS library):
```js
const subscription = supabase
  .from('*')
  .on("*", (payload) => { console.log(payload) })
  .subscribe((status) => { console.log("Subscription status: ", status) });

console.log(subscription);
```

Console output:
![image](https://user-images.githubusercontent.com/10406383/193955439-b3238e6a-e4b5-418b-a2c7-3dbadcfaba04.png)

Websocket logs:
![image](https://user-images.githubusercontent.com/10406383/193955494-e82d095c-f5fb-4937-af74-6211565dcab3.png)

And most notably based on some comments from [Discussion #428](https://github.com/supabase/supabase/discussions/428), here's the output when I run `SELECT * FROM pg_replication_slots;`
![image](https://user-images.githubusercontent.com/10406383/193955698-2908d709-aac2-405d-804d-824041c5f9dc.png)

I made a new project and was able to successfully get Realtime updates working with the exact same code, so I'm fairly sure it's not my code. My question is, how can I remedy the issue with my current project so that I don't have to migrate all my data to a new project?

**Comment by GaryAustin1** [comment, marked as answer]
Please contact support.  There were a couple of issues over the past 2 weeks with certain projects and upgrading to the new realtime servers.

**Reply by CactusPuppy** [reply]
Will do, thanks!
